libs/core/itt_notify/CMakeLists.txt

```diff
@@ -20,13 +20,17 @@ set(itt_notify_compat_headers
 
 set(itt_notify_sources itt_notify.cpp thread_name.cpp)
 
+if(HPX_WITH_ITTNOTIFY)
+  set(itt_notify_dependencies DEPENDENCIES Amplifier::amplifier)
+endif()
+
 include(HPX_AddModule)
 add_hpx_module(
   core itt_notify
   GLOBAL_HEADER_GEN OFF
   SOURCES ${itt_notify_sources}
   HEADERS ${itt_notify_headers}
-  COMPAT_HEADERS ${itt_notify_compat_headers}
+  COMPAT_HEADERS ${itt_notify_compat_headers} ${itt_notify_dependencies}
```

Comment:
```suggestion
  COMPAT_HEADERS ${itt_notify_compat_headers}
  DEPENDENCIES ${itt_notify_dependencies}
```

Reply:
Can the list of dependencies be empty?

Reply:
Ah, I see now why `DEPENDENCIES` was "missing". However, it can be empty and I find it clearer if the keyword is in the call and not in the variable. That's just my preference though.

Reply:
I agree with having it in the call, will change.